modules/cluster-logging-deploy-clo.adoc

```diff
@@ -1,86 +1,27 @@
 // Module included in the following assemblies:
 //
-// * logging/cluster-logging-deploy.adoc
+// * logging/cluster-logging-deploying.adoc
 
-[id="cluster-logging-deploy-subscription_{context}"]
-= Installing the Cluster Logging and Elasticsearch Operators
+[id="cluster-logging-deploy-clo_{context}"]
+= Install the Cluster Logging Operator using the Console
 
-You can use the {product-title} console to install cluster logging, by deploying,
-the Cluster Logging and Elasticsearch Operators.  The Cluster Logging Operator
-creates and manages the components of the logging stack.  The Elasticsearch Operator
-creates and manages the Elasticsearch cluster used by cluster logging.
+You can use the {product-title} console to install the Cluster Logging Operator.  
 
-[NOTE]
-====
-The {product-title} cluster logging solution requires that you install both the
-Cluster Logging Operator and Elasticsearch Operator. There is no use case
-in {product-title} for installing the operators individually.
-You *must* install the Elasticsearch Operator using the CLI following the directions below.
-You can install the Cluster Logging Operator using the web console or CLI.
-====
+. Create a Namespace for the Cluster Logging Operator:
 
-.Prerequisites
+.. In the {product-title} web console, click *Home* -> *Projects*.
```

Comment:
I'd suggest being consistent about using "web console" or just "console" (not positive, but I think the more common convention is "web console").

Reply:
We seem to use both `{product-title} console` and `{product-title} web console`. `{product-title} web console` is much more prevalent.

Reply:
outdated